keel-core/src/main/kotlin/com/netflix/spinnaker/keel/telemetry/TelemetryEvent.kt

```diff
@@ -50,3 +52,8 @@ data class ArtifactCheckTimedOut(
   val name: String,
   val deliveryConfigName: String?
 ) : TelemetryEvent()
+
+data class ArtifactSaved(
```

Comment:
Is this really a `TelemetryEvent`? My guess was that those were only events that we were using to collect metrics.

Reply:
it's not. I just follow the pattern but you're right, it not a `TelemetryEvent`. removed